lib/errors.js

```diff
@@ -57,19 +43,14 @@ function defaultErrorFormatter (execution, ctx) {
   const errors = execution.errors.map((error) => {
     log.info({ err: error }, error.message)
 
-    // parses, converts & combines errors if they are the result of a federated request
-    if (error.message === FEDERATED_ERROR.toString() && error.extensions) {
-      return error.extensions.errors.map(err => toGraphQLError(err))
-    }
```

Comment:
Are we overriding the error handler in the gateway?

Reply:
Yes, the gateway replace the errorHandler, if not passed as opts. And the gateway handler has the check above.